src/Entity/OgRole.php

```diff
@@ -187,6 +222,10 @@ public function save() {
         throw new ConfigValueException('The group bundle can not be empty.');
       }
 
+      if (empty($this->getName())) {
+        throw new ConfigValueException('The role name can not be empty.');
```

Comment:
I had a look at this but it is not really possible I think. The `DefaultRoleEvent::validate()` method is used to validate only the data that is necessary for the event listener to gather the default roles. This one is in `OgRole::save()` and validates everything required to successfully store the entity.

These are different use cases, and calling `validate()` from here will make `OgRole` depend on `DefaultRoleEvent`.